less/variables.less

```diff
@@ -1,5 +1,7 @@
 // Variables
 
+@desktop: "(max-width: 800px)";
```

Comment:
to do escaping, you would need the ~ line in front of the string.  that is escape syntax.